src/js/media-container.js

```diff
@@ -18,6 +19,7 @@ template.innerHTML = `
     :host {
       box-sizing: border-box;
       position: relative;
+      align-items: stretch;
```

Comment:
Did this fix something specific?

Reply:
yes. It ensures that the newly-added landmark `section` sizes to the parent for both dimensions.